test/test_io.py

```diff
@@ -184,6 +184,13 @@ def test_read_video_pts_unit_sec(self):
             self.assertEqual(info["video_fps"], 5)
             self.assertEqual(info, {"video_fps": 5})
 
+    def test_read_video_pts_out_of_stream(self):
+        with temp_video(10, 300, 300, 5, lossless=True) as (f_name, data):
+            with self.assertRaises(BufferError):
+                lv, _, info = io.read_video(f_name, start_pts=5, pts_unit='sec')
+        with temp_video(10, 300, 300, 5, lossless=True) as (f_name, data):
```

Comment:
Do we need this check given that we do test for correct number of frames in other tests?

Reply:
Do you mean the one in L191? Just wrote it to double check there were no issues with standard loading since I didn't checked all the test